app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -299,6 +300,9 @@ private void rethrowResourceException(ResourceException e, Account account) {
         case 7104: // Account not found in the directory
         case 2016: // Property value does not match a known resource. Somehow this equals not found.
             throw new EntityNotFoundException(Account.class);
+        case 7101:
+            // Account is disabled for administrative reasons. This throws 423 LOCKED (WebDAV, not pure HTTP)
+            throw new BridgeServiceException("Account disabled, please contact user support", HttpStatus.SC_LOCKED);
```

Comment:
We should add 423 response to the API docs, noting this is something that needs to be communicated back to us in order to resolve.